### DISCUSSION TITLE
Seurat CCA vs rlsi prior to ATAC integration

### DISCUSSION THREAD

**Discussion by LacquerHed** [GreenleafLab/ArchR#1663] [Questions / Documentation]
I was wondering whether using the rlsi reduction during Seurat scRNA-seq integration is most appropriate when considering downstream integration with scATAC-seq in ArchR? Or is CCA also ok? Thanks.

**Comment by rcorces** [comment]
I dont believe that any of that has an effect on integration. Integration is based on the counts in the RNA matrix, not on the dim red. of the RNA.
